book/content/version_control/vcs-github.md

```diff
@@ -0,0 +1,184 @@
+## Using Git commands to work on GitHub
+
+As research becomes increasingly collaborative and multiple people work on the same project, it becomes difficult to keep track of changes made by others if not done systematically.
+Moreover, it is time-consuming to manually incorporate the work of different participants in a project, even when all of their changes are compatible.
+Hosting the project on an online distributed version control system like GitHub is beneficial to make collaborations open and effective.
+If you are new to collaboration through [GitHub](https://github.com), please follow a comprehensive guide in the collaboration section.
+
+In this section we will discuss how to use Git commands to work on GitHub repository.
+
+### Create a local copy of an online repository
+
+There are some GitHub features that can be accessed via command-line Git tool.
```

Comment:
Just as a comment: In principle, none of these commands are specific to GitHub. Cloning, Git remotes, pushing, pulling is part of the standard Git tool kit, and were originally (and are still) used for collaborative work on Git repositories. Therefore, all of these commands are not specifically created for interactions with GitHub, but rather with interactions on any repository hosting site/server (which can be GitHub, but also GitLab, Bitbucket, Gin, or a self-set-up bare Git repository on a web server). I do not think, however, that all of this background is useful to add here at this point of the turing way and rather distracts from the main point in this chapter - which is: how do I use it efficiently. But I'm adding this comment just in case and to keep in mind for more advanced chapters.

Reply:
That's a good point @adswa. I think we should add a note here with details you have listed in your comment. We can say that for simplicity purpose, we will use GitHub as a platform to explain these commands. Do you think it will help clarify?